src/common/transformations/src/transformations/sdpa_to_paged_attention/state_management_pattern.cpp

```diff
@@ -544,17 +547,25 @@ ov::pass::StateManagementPattern::StateManagementPattern(ParameterVector& kv_par
             block_indices_inputs_for_each_layer.push_back(block_indices);
         }
 
-        OPENVINO_ASSERT(pa_arguments.size() == 13);
+        if (allow_score_aggregation) {
+            OPENVINO_ASSERT(optional_model_wide_params.find("score_aggregation_window") !=
```

Comment:
could you provide additional details in the assertion message, e.g. 
"No score_aggregation_window input found. For using score aggregation mode, the model have to contain an additional input (Parameter) called score_aggregation_window."

Reply:
Done